tests/unit/test_clidocs.py

```diff
@@ -406,6 +407,29 @@ def test_includes_global_args_ref_in_html_options(self):
             "global parameters", rendered
         )
 
+    def test_includes_streaming_blob_in_man_options(self):
+        help_command = self.create_help_command()
+        blob_shape = Shape('blob_shape', {'type': 'blob'})
+        blob_shape.serialization = {'streaming': True}
+        blob_arg = CustomArgument('blob_arg', argument_model=blob_shape)
+        help_command.arg_table = {'blob_arg': blob_arg}
+        operation_handler = OperationDocumentEventHandler(help_command)
+        operation_handler.doc_option(arg_name='blob_arg', help_command=help_command)
+        rendered = help_command.doc.getvalue().decode('utf-8')
+        self.assertIn('streaming blob', rendered)
+
+    def test_includes_streaming_blob_in_html_options(self):
```

Comment:
Similar to another PR, I'm not sure if we need to add another test for when html is the target especially since there is no branching logic. I'd suggest just removing this test and rename the other one to `test_includes_streaming_blob_options()`.